apps/sr-frontend/src/app/App.tsx

```diff
@@ -53,7 +53,7 @@ function Content() {
     >
       <Header anchor="back-to-top-anchor" />
 
-      <Container maxWidth="xl" sx={{ px: { xs: 0.5, sm: 1 } }}>
+      <Container maxWidth="xl" sx={{ px: { xs: -1.5, sm: 1 } }}>
```

Comment:
Not sure we need this. We can also fix this later

Reply:
Seems like this was done with the merge.

Reply:
Can you undo it? Seems like a bad merge somehow, should just keep it as 0.5 padding

Reply:
done